R/arx_classifier.R

```diff
@@ -171,11 +190,25 @@ arx_class_epi_workflow <- function(
         )
     }
   }
-  o2 <- rlang::sym(paste0("ahead_", args_list$ahead, "_", o))
+  ahead_out_name <- glue::glue("ahead_[0-9]*_{pre_out_name}")
```

Comment:
Can you explain what this should do? I see:
``` r
pre_out_name <- "pre_out_name" # just to examine
glue::glue("ahead_[0-9]*_{pre_out_name}")
#> ahead_[0-9]*_pre_out_name
```

Reply:
oh this is a regex that catches anything that looks like `ahead_<any set of numbers>_<pre_out_name>`

Reply:
Is it missing some data to key on? unless I'm missing something, it just prints the regex, not the result of the regex.

Reply:
`ahead_out_name` is used in a `matches`, which uses regex. Its in a separate variable to avoid execution shenanigans. renaming to make it clear it should just be a regex

Reply:
I understand how it gets used. My concern is: 
```r
 glue::glue("ahead_[0-9]*_{pre_out_name}")
```

Why is `[0-9]*` hardcoded into the name? Glue doesn't reverse regex it. Did you mean this to be `{ahead}`? 
Whereas (if regex matching) later on you can use `[0-9]*` to match the result (in the `ahead_out_name` object)?

Reply:
`ahead` may not be the actual name of the column, because that value could change b/c of ahead adjustment. I'm not expecting glue to do anything with `[0-9]*` but pass it on, since `ahead_out_name` is a regex (which are just strings). `ahead_[0-9]*_pre_out_name` is the expected result for `ahead_out_name_regex`.